src/document.js

```diff
@@ -545,6 +545,7 @@ const documentsMain = {
 					PostMessages.sendWOPIPostMessage('loolframe', 'Disable_Default_UIAction', { action: 'UI_Hyperlink', disable: true })
 				}
 			})
+			PostMessages.sendWOPIPostMessage('loolframe', 'Error_Messages', { websocketconnectionfailed: 'Failed to establish socket connection or socket connection closed unexpectedly. The reverse proxy might be misconfigured, please contact the administrator. For more info on proxy conifguration please checkout https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md'})
```

Comment:
Is there a post message also emitted back to Nextcloud on error messages? It might be nice to actually show this during the loading screen already.

Documentation link might need some adjustment as all in one is just way of setting up.

Reply:
> Is there a post message also emitted back to Nextcloud on error messages? It might be nice to actually show this during the loading screen already.

@juliushaertl we can send 'Action_Load_Resp' to show message in nextcloud overlay
checkout this PR: https://github.com/CollaboraOnline/online/pull/7375

> Documentation link might need some adjustment as all in one is just way of setting up.
what should be appropriate link ?

Reply:
Probably https://docs.nextcloud.com/server/latest/admin_manual/office/proxy.html which just links to the Collabora SDK docs